### DISCUSSION TITLE
v8.01 - getting it to work

### DISCUSSION THREAD

**Discussion by antamy** [tagyoureit/nodejs-poolController#724] [App and O/S Config Questions] [answered]
I did a fresh install of v8.01, after archiving my v7.x install.  Modified config.json to have the correct /dev/tty for the RS485 adaptor.   It all starts up, but I just get a Nixie Single Body system with no equipment.

If I replace poolConfig.json and poolState.json with my working config from v7, everything is displayed correctly, however my chlorine %ge is incorrect.  Before reporting a bug on that, am I doing anything wrong here??

**Comment by rstrouse** [comment, marked as answer]
Please use the dashPanel UI to edit the port.  A Single Body Nixie controller when you have a Pentair OCP is indicative of a port that is not capturing the data from the panel.  dashPanel will show the packet failures and response and display any errors when attempting to open the port.

**Reply by antamy** [reply]
Some very odd behavior being seen.  I am filing a bug report.
